packages/select/test/accessibility.test.js

```diff
@@ -1,13 +1,16 @@
 import { expect } from '@esm-bundle/chai';
-import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
+import { fixtureSync, nextFrame, nextRender } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import '@vaadin/item/vaadin-item.js';
 import '@vaadin/list-box/vaadin-list-box.js';
 import './not-animated-styles.js';
 import '../vaadin-select.js';
 
 describe('accessibility', () => {
-  let select, valueButton;
+  /**  @type {HTMLElement} */
```

Comment:
We don't really need types in tests, but feel free to keep them. Not a blocker.

Reply:
I added them to help in the autocomplete. Removed.